pkg/kube/kube.go

```diff
@@ -150,45 +92,24 @@ func WaitForPodsWithLabelRunning(c kubernetes.Interface, ns string, label labels
 
 		for _, pod := range pods.Items {
 			if pod.Status.Phase != core.PodRunning {
+				glog.Infof("temporary error: for Pod with label %q expected status to be running but got %s : [%v]\n", label.String(), pod.Status.Phase, err)
```

Comment:
I'm not sure this is an error, and I also find this difficult to read. Could this be reworded with less words? Perhaps just:

"waiting for pod %q, current state: %s"